src/components/Manager/ManagerHeader.tsx

```diff
@@ -103,14 +103,14 @@ const ManagerHeader = ({
       );
     }
 
-    setFilteredDemands(filteredDemands);
+    onFilterUpdate(filteredDemands);
   }, [
     addressFilter,
     nameFilter,
     filterModeChauffage,
     filterTypeChauffage,
     demands,
-    setFilteredDemands,
```

Comment:
I find this change pretty aggressive as i explicitely refused to do it :/

Reply:
I'm sorry you're seeing an aggressive update because that's absolutely not the goal. And also, this is not for the purpose for opposite the code you had provided. 
In this feature I am passing a complete function to call after update the filter and not just an update status so in my mine this is more appropriate. (And when i have submit this name, in your PR, it is in the projection of this)
On this line, it's just the name of the function in the props. If you prefer, I can go back to using your original "setFilteredDemands" name.